modules/nw-dpu-creating-a-sfc.adoc

```diff
@@ -28,17 +28,30 @@ metadata:
   name: sfc
   namespace: openshift-dpu-operator
 spec:
+  nodeSelector:
+    dpu.config.openshift.io/dpuside: "dpu"
   networkFunctions:
-  - name: my-network-function <1>
-    image: quay.io/example-org/my-network-function:latest <2>
+  - name: my-network-function
+    image: quay.io/example-org/my-network-function:latest
 ----
 +
-<1> The name of the network function. This name is used to identify the network function in the service function chain.
-<2> The URL to the container image that contains the network function. The image must be accessible from the DPU.
+* `spec.nodeSelector`: The `nodeSelector` is the primary mechanism for scheduling this workload.
+** The label `dpu.config.openshift.io/dpuside: "dpu"` is created and maintained by the DPU Operator. This label ensures the pod is scheduled directly onto the DPU's processing unit.
+* `spec.networkFunctions.name`: The name of the network function. This name is used to identify the network function in the service function chain.
+* `spec.networkFunctions.iamge`:The URL to the container image that contains the network function. The image must be accessible from the DPU.
```

Comment:
Typo should be spec.networkFunctions.image but we should change this to spec.containers.image